content/400-reference/200-api-reference/300-environment-variables-reference.mdx

```diff
@@ -0,0 +1,135 @@
+---
+title: 'Environment variables reference'
+metaTitle: 'Prisma environment variables reference'
+metaDescription: 'This page gives an overview of all environment variables avalaible for use.'
+tocDepth: 2
+---
+
+## Overview
+
+This document describes different environment variables and their use cases.
+
+See [Connection URLs](../database-reference/connection-urls) for a breakdown of each database connector.
+
+### DEBUG
+
+`DEBUG` is used to enable debugging output in the Prisma Client.
+
+Example setting Prisma Client level debugging output:
+
+```terminal
+# enable only `prisma-client`-level debugging output
+export DEBUG="prisma-client"
+```
+
+See [Debugging](../../concepts/components/prisma-client/debugging) for more information.
+
+### NO_COLOR
+
+`NO_COLOR` if [truthy](https://developer.mozilla.org/en-US/docs/Glossary/Truthy) will activate the `colorless` setting for error formatting and strip colors from error messages.
+
+See [Formatting via environment variables](../../concepts/components/prisma-client/working-with-prismaclient/error-formatting#formatting-via-environment-variables) for more information.
+
+### NO_PROXY
+
+`NO_PROXY` is a comma-seperated list of hostnames, IP addresses or IP ranges that do not require a proxy.
```

Comment:
This could use explanation where this is used.

Reply:
Im not 100% sure where it is used. I see it in the code for the fetch-engine package. We dont currently document a use case in the docs. If this is just used internally then we dont need to tell users about it..

Reply:
It's used in fetch-engine when we download binaries inside `getProxyAgent` https://github.com/prisma/prisma/blob/aac667e011979fd13772fcbbb9ae4bc79251d158/src/packages/fetch-engine/src/getProxyAgent.ts#L94

Reply:
But thats internal right? Do you think we should document it?

Reply:
I tried to improve the why here https://github.com/prisma/docs/pull/1422/files#r596665691, I think we can keep it.